man/paru.conf.5

```diff
@@ -89,49 +91,48 @@ PKGBUILD.
 .B CombinedUpgrade
 During sysupgrade, paru will first perform a refresh, then show
 its combined menu of repo and AUR packages that will be upgraded. Then after
-reviewing the pkgbuilds, the repo and AUR upgrade will start with no need
+reviewing the PKGBUILDs, the repo and AUR upgrade will start with no need
 for manual intervention.
 
-If paru exits for any reason After the refresh without upgrading. It is then
+If paru exits for any reason after the refresh without upgrading, it will be
 the user's responsibility to either resolve the reason paru exited or run
 a sysupgrade through pacman directly.
 
 .TP
 .B BatchInstall
-When building and installing AUR packages instead of installing each package
+When building and installing AUR packages; instead of installing each package
 after building, queue each package for install. Then once either all packages
 are built or a package in the build queue is needed as a dependency to build
 another package, install all the packages in the install queue.
 
 .TP
 .B UseAsk
-Use pacman's --ask flag to automatically confirm package conflicts. paru lists
+Use pacman's --ask flag to automatically confirm package conflicts. Paru lists
 conflicts ahead of time. It is possible that paru does not detect
 a conflict, causing a package to be removed without the user's confirmation.
 However, this is very unlikely.
 
 .TP
-.B Redownload [= All]
```

Comment:
Again `[...]` means optional. So `Redownload` on it's own means yes `Redownload = All` for all.

Looking at the code, `Redownload = No` works cause it uses the same validator as --redownload. But using it doesn't make sense and I may remove it. so lets not advertise it.